package.json

```diff
@@ -51,7 +51,7 @@
     "pinst": "^2.1.6"
   },
   "engines": {
-    "node": "14.x"
+    "node": "10.19.0"
```

Comment:
Let's not commit changes that are not related to the issue.
Can we revert this change?